src/Plugin/QueueWorker/CronEncryptedFieldUpdate.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\field_encrypt\Plugin\QueueWorker\CronEncryptedFieldUpdate.
+ */
+
+namespace Drupal\field_encrypt\Plugin\QueueWorker;
+
+use Drupal\field_encrypt\Plugin\QueueWorker\EncryptedFieldUpdateBase;
+
+/**
+ * A Queue Worker that updates field encryption on cron run.
```

Comment:
This could indeed lag the updates, so it would probably be best to add advice to run the updates manually whenever possible.

Normally a manual run and a cronjob wouldn't interfere with each other, since they won't process an item that's claimed by the other...

I thought it would be advisable to have a backup solution with cron, for cases where the manual process can't be run, or where slower encryption updates are not a real issue. 

Either way a queued approach is always tricky, especially in changing environments, but I don't see a lot of alternatives. In my latest commits I have updated the Drupal message being shown, and the readme file, to warn about the potential consequences of this process, and the urgency to perform it when settings have been changed.